### PR TITLE
populate the instructions on installing kubectl in getting-started-guides

### LINKED ISSUE #9632: Add link to kubectl.md in Getting Started Guide
Suggest we link to https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/kubectl.md in the getting started guides.  Anyone standing up their first cluster is going to want to play around with all the kubectl options, and this is a useful reference page.

### PR DESCRIPTION
**Pull request by caesarxuchao**
Fixes #9632 and some nits in getting-started-guides

Regarding this [comment](https://github.com/GoogleCloudPlatform/kubernetes/issues/9632#issuecomment-120567421), because there is no TOC page for docs/user-guide, so I followed the previous [comment](https://github.com/GoogleCloudPlatform/kubernetes/issues/9632#issuecomment-119004568)

@nikhiljindal @JanetKuo

### CODE REVIEW

**Comment by caesarxuchao** [comment]
I will address this [comment](https://github.com/GoogleCloudPlatform/kubernetes/pull/10595#discussion_r34642152) in this PR.

**Comment by bgrant0607** [comment]
cc @erictune

File: docs/getting-started-guides/aws-coreos.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ no security tokens, no basic auth). For demonstration purposes only.
 
 * [aws CLI](http://aws.amazon.com/cli)
 * [CoreOS image for AWS](https://coreos.com/docs/running-coreos/cloud-providers/ec2/)
-* [kubectl CLI](aws/kubectl.md)
+* [kubectl CLI](aws/kubectl.md) ([installation](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/getting-started-guides/aws.md#command-line-administration-tool-kubectl))
```
Comment (bgrant0607) [inline docs/getting-started-guides/aws-coreos.md]: Must use relative links.

Comment (bgrant0607) [inline docs/getting-started-guides/aws-coreos.md]: Also, why does AWS have its own instructions for installing kubectl?

Comment (bgrant0607) [inline docs/getting-started-guides/aws-coreos.md]: Nevermind the latter comment.

**Comment by bgrant0607** [comment]
@jlowdermilk @deads2k PTAL

**Comment by bgrant0607** [comment]
The user guide TOC is in docs/user-guide/README.md, but I would link directly to docs/user-guide/prereqs.md.

File: docs/user-guide/prereqs.md [hunk 2]
```diff
@@ -21,11 +21,30 @@ certainly want the docs that go with that version.</h1>
...
+Next, make sure the kubectl tool is in your path, assuming you download a release:
+```
+# OS X
+export PATH=<path/to/kubernetes-directory>/platforms/darwin/amd64:$PATH
+
+# Linux
+export PATH=<path/to/kubernetes-directory>/platforms/linux/amd64:$PATH
+```
+
+An alternate method, useful if you're building from source and do not want to configure the PATH, is using ./cluster/kubectl.sh instead of kubectl. That script will auto-detect the location of kubectl and proxy commands to it (eg., ./cluster/kubectl.sh cluster-info).
```
Comment (jlowdermilk) [inline docs/user-guide/prereqs.md]: Would prefer we not direct users to this method. We want to stop using cluster/kubectl.sh as a command proxy and instead have it find the binary location and `export PATH=...`

Comment (bgrant0607) [inline docs/user-guide/prereqs.md]: Bonus points if someone can better organize and cross-refernece the kubectl installation and kubeconfig docs, including this one, the text in the getting-started guides and examples (e.g., docs/user-guide/walkthrough), docs/user-guide/accessing-the-cluster.md, docs/user-guide/sharing-clusters.md, ...

Comment (bgrant0607) [inline docs/user-guide/prereqs.md]: Agree we shouldn't mention kubectl.sh, btw. See #8817

File: docs/user-guide/prereqs.md [hunk 3]
```diff
@@ -21,11 +21,30 @@ certainly want the docs that go with that version.</h1>
...
+
+##Configure kubectl
 In order for kubectl to find and access the Kubernetes cluster, it needs a [kubeconfig file](kubeconfig-file.md), which is created automatically when creating a cluster using kube-up.sh (see the [getting started guides](../../docs/getting-started-guides/) for more about creating clusters). If you need access to a cluster you didn’t create, see the [Sharing Cluster Access document](sharing-clusters.md).
 
+Check that kubectl is properly configured by getting the cluster state:
+```
+kubectl cluster-info
```
Comment (bgrant0607) [inline docs/user-guide/prereqs.md]: Put a `$` at the beginning.

**Comment by caesarxuchao** [comment]
I've pushed a commit to address the comments. 

@bgrant0607 I'm not sure what this [comment](https://github.com/GoogleCloudPlatform/kubernetes/pull/11276#issuecomment-121712068) is about.

**Comment by bgrant0607** [comment]
LGTM. Please squash.

**Comment by caesarxuchao** [comment]
Squashed. Thanks.

**Comment by jlowdermilk** [comment]
LGTM

**Pull request merged**